deps/datReader/Exd.cpp

```diff
@@ -279,106 +233,100 @@ namespace xiv::exd
   }
 
   // Get all rows
-  const std::map< uint32_t, std::vector< Field>>& Exd::get_rows()
+  const std::map< double, std::vector< Field > > Exd::get_rows()
```

Comment:
yeah, uh, lets not do this lol, this is bound to have some bullshit issue because of floating point inaccuracy

just have a u64 and treat the low/high u32 as the row/subrow

Reply:
or even use a struct and abuse a union to give you a u64 and 2 u32s

Reply:
The floating point inaccuracy wasn't really a problem in testing, but as you said it could create problems in the future.
Unfortunately unions aren't working as a key in containers. So I will went with your first approach.

Reply:
you will need to add a custom hash function and then it'll work

Reply:
https://en.cppreference.com/w/cpp/utility/hash

Reply:
but you can just pass it the u64 value directly

Reply:
I could do that, but I don't wanted to make it more complex as necessary

Reply:
it is generally helpful to be able to access the row ids easily, so it being simpler here makes it more complicated elsewhere ;)

Reply:
yeah, you're right 👍